roles/discover/tests/test.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+- hosts: localhost
+  roles:
+    - galaxy/discovery/roles/discover
```

Comment:
This did not resolve properly. When `galaxy/discovery/` was removed, the role executed successfully